util/migration-rocksdb/src/lib.rs

```diff
@@ -331,7 +324,8 @@ impl Progress {
 		self.current += 1;
 		if self.current == self.max {
 			self.current = 0;
-			flush!(".");
+			// TODO(niklasad1): check why we wrote `.` to stdout before seems needless
```

Comment:
thoughts?

Reply:
It's like a poor man's progress indicator I think, at least that's how I read it. Imo it can go.

Reply:
exactly, we can use a proper progress bar lib like https://github.com/mitsuhiko/indicatif or whatever cool kids are using these days

Reply:
It was not used, so I removed it